feature/google/build.gradle

```diff
@@ -28,5 +28,6 @@ dependencies {
     testImplementation "org.assertj:assertj-core:$assertJVersion"
     testImplementation 'com.google.android.gms:play-services-ads-identifier:17.0.1'
     testUnityIAPImplementation "com.android.billingclient:billing:$billing4Version"
+    testBc5SupportDevImplementation "com.android.billingclient:billing:$billing5Version"
```

Comment:
Hmm is this needed? We are adding this library using `bc5SupportDevImplementation` so it should also be available in tests...

Reply:
oh, maybe not...then that would mean we don't need the `testUnityIapImplementation` either, right? at least not until we merge all the bc5 things? i'd maybe leave it anyway just so we don't forget, i'm just making sure i'm understanding..

Reply:
I believe the `unityIAP` one might still be needed since that is using `compileOnly`. That should be the main difference between `unityIAP` and `latestDependencies`

Reply:
ahhh yes, thank you. makes sense